examples/bulk-ingest/bulk-ingest.py

```diff
@@ -6,13 +6,14 @@
 """Script that downloads a public dataset and streams it to an Elasticsearch cluster"""
 
 import csv
-from os.path import abspath, join, dirname, exists
+from os.path import abspath, dirname, exists, join
```

Comment:
Please use `nox -rs format` to avoid unrelated changes like this. While we should apply isort/black to those files, it should be in a separate pull request.